publisher/standard/README.md

```diff
@@ -1,6 +1,6 @@
 # UID2 SDK Integration Example
 
-[This example](https://example-jssdk-integ.uidapi.com/) demonstrates how a content publisher can use the [UID2 services](https://github.com/UnifiedID2/uid2docs/tree/main) and the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md) (also known as the UID2 SDK) to implement the [standard UID2 integration workflow](https://unifiedid.com/docs/guides/publisher-client-side).
+[This example](https://example-jssdk-integ.uidapi.com/) demonstrates how a content publisher can use the [UID2 services](https://unifiedid.com/docs/intro) and the [Client-Side Identity JavaScript SDK](https://unifiedid.com/docs/sdks/client-side-identity) (also known as the UID2 SDK) to implement the [standard UID2 integration workflow](https://unifiedid.com/docs/guides/publisher-client-side).
```

Comment:
[This example](https://example-jssdk-integ.uidapi.com/) demonstrates how a content publisher can use the [UID2 services](https://unifiedid.com/docs/intro) and the [Client-Side JavaScript SDK](https://unifiedid.com/docs/sdks/client-side-identity) to implement the [standard UID2 integration workflow](https://unifiedid.com/docs/guides/publisher-client-side).

Remove "Identity" from the SDK title. But I think that "also known as the UID2 SDK" is probably not needed? Perhaps old copy from when there weren't so many? Seems unnecessary.